Std/Data/Nat/Lemmas.lean

```diff
@@ -344,7 +351,7 @@ protected theorem add_self_ne_one : ∀ (n : Nat), n + n ≠ 1
     have h1 : succ (succ (n + n)) = 1 := succ_add n n ▸ h
     Nat.noConfusion h1 fun.
 
-/-! ### sub -/
+/-! ## sub -/
```

Comment:
I think it makes sense to keep the headers with the same size as everywhere else even if we're not using the intermediate size.

Reply:
I don't think we are being very consistent about header size. It probably requires a global review.

Reply:
At least, we're consistent within mathlib.

Reply:
Should I change from `##` to `###`? The file was inconsistent so I picked one. I have no personal preference for 2 or 3.

Reply:
I'd personally go for `###` but I guess Mario might have a different opinion.